### PR TITLE
Update Moderation Policy to reflect TSC charter changes

### PR DESCRIPTION
**Pull request by Trott**
Ref: https://github.com/nodejs/TSC/pull/1350

@nodejs/tsc

### CODE REVIEW

File: Moderation-Policy.md [hunk 1]
```diff
@@ -257,15 +257,15 @@ Moderation team members have the same expectations as other leadership groups
 as outlined [here](https://github.com/nodejs/admin/blob/master/MemberExpectations.md).
 
 At least once per month, the Moderation Team must provide a report of all Moderation
-actions taken by the Moderation Team to the TSC.
+actions taken by the Moderation Team to TSC voting members.
```
Comment (aduh95) [inline Moderation-Policy.md:260]:
```suggestion
actions taken by the Moderation Team to the TSC.
```

File: Moderation-Policy.md [hunk 2]
```diff
@@ -257,15 +257,15 @@ Moderation team members have the same expectations as other leadership groups
...
 
 *Nomination*
 
 Moderation team members are Collaborators who self-nominate or are nominated by 
-the TSC. Team members must be approved by the TSC with annual 
+the TSC. Team members must be approved by the TSC voting members with annual
```
Comment (aduh95) [inline Moderation-Policy.md:265]:
```suggestion
the TSC. Team members must be approved by the TSC with annual 
```

File: Moderation-Policy.md [hunk 3]
```diff
@@ -324,8 +324,8 @@ remove resigning team member from respective permissions and private access.
 
 ## Escalation of Issues
 
-Moderation issue disputes not involving a TSC or Moderation Team
-member may be escalated to the TSC for review by tagging the
+Moderation issue disputes not involving a TSC voting member or Moderation Team
+member may be escalated to the TSC voting members for review by tagging the
```
Comment (aduh95) [inline Moderation-Policy.md:328]:
```suggestion
member may be escalated to the TSC for review by tagging the
```

File: Moderation-Policy.md [hunk 4]
```diff
@@ -264,8 +264,8 @@ actions taken by the Moderation Team to the TSC.
 Moderation team members are Collaborators who self-nominate or are nominated by 
 the TSC. Team members must be approved by the TSC with annual 
 recertification. If there are no objections after seven days, the nomination is automatically
-accepted. If there are objections to a specific nomination, then a simple majority
-vote of the TSC in favor of the nomination is required.
+accepted. If there are objections to a specific nomination, then TSC vote
```
Comment (Trott) [inline Moderation-Policy.md:267]:
```suggestion
accepted. If there are objections to a specific nomination, then a TSC vote
```

**Review by targos — APPROVED** [review]

**Review by aduh95 — APPROVED** [review]

**Review by GeoffreyBooth — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Review by mhdawson — COMMENTED** [review]
LGTM 
EDIT: just noticed already landed :)

**Pull request merged**